WordPress/Classes/ViewRelated/Stats/Shared Views/Date Chooser/SiteStatsTableHeaderView.swift

```diff
@@ -311,3 +320,25 @@ extension SiteStatsTableHeaderView: StatsBarChartViewDelegate {
         reloadView()
     }
 }
+
+private extension SiteStatsTableHeaderView {
+    enum Metrics {
+        static let dateLabelFontSize: CGFloat = 20
+        static let maximumDateLabelFontSize: CGFloat = 32
+        static let timezoneFontSize: CGFloat = 16
+        static let maximumTimezoneFontSize: CGFloat = 20
+        static let minimumScaleFactor: CGFloat = 0.8
+
+        static var dateLabelFont: UIFont {
+            let fontDescriptor = UIFontDescriptor.preferredFontDescriptor(withTextStyle: .headline)
+            let font = UIFont(descriptor: fontDescriptor, size: dateLabelFontSize)
+            return UIFontMetrics.default.scaledFont(for: font, maximumPointSize: maximumDateLabelFontSize)
+        }
+
+        static var timezoneFont: UIFont {
+            let fontDescriptor = UIFontDescriptor.preferredFontDescriptor(withTextStyle: .callout)
+            let font = UIFont(descriptor: fontDescriptor, size: timezoneFontSize)
+            return UIFontMetrics.default.scaledFont(for: font, maximumPointSize: maximumTimezoneFontSize)
+        }
+    }
+}
```

Comment:
@staskus nice + I like this encapsulated 👍

Reply:
Thanks 👍